src/api/CMakeLists.txt

```diff
@@ -39,7 +39,6 @@ include_directories(
 # Mercury
 find_package(MERCURY REQUIRED)
 if(MERCURY_FOUND)
-  message(STATUS "mercury dir ${MERCURY_DIR}")
```

Comment:
Is this dumped in FindMercury? If not, let's make sure the path is shown somewhere.

Reply:
This is already dumped in FindMercury.cmake. That's why I'm removing it.